aws/asg/README.md

```diff
@@ -21,13 +21,12 @@ an auto scaling group that uses the configuration.
 ## Optional Inputs
```

Comment:
Should optional inputs be included in the "Example Usage" section?

Reply:
eh, I usually leave them out for the simplest usage example